packages/calcite-components/src/components/action-bar/action-bar.tsx

```diff
@@ -345,6 +351,18 @@ export class ActionBar
     this.setGroupLayout(groups);
   };
 
+  handleActionsEndSlotChange = (event: Event): void => {
+    this.hasActionsEnd = slotChangeHasAssignedElement(event);
+  };
+
+  handleTooltipSlotChange = (event: Event): void => {
+    const tooltips = slotChangeGetAssignedElements(event).filter((el) =>
+      el?.matches("calcite-tooltip")
```

Comment:
Just curious, when would this be `null`?

Reply:
are you talking about `el`? If there are no assigned elements.

Reply:
> If there are no assigned elements.

If it returns an empty array for this case, the `filter` callback won't be invoked.